python/lsst/pipe/base/pipelineTask.py

```diff
@@ -309,24 +364,47 @@ def runQuantum(self, quantum, butler):
                 inputs[key] = [butler.get(dataRef.datasetType.name, dataRef.dataId)
                                for dataRef in dataRefs]
 
```

Comment:
There is no good way to put a comment in a non diff line, so I am putting this here. The changes made require that the documentation in the docstring for run quantum need to be updated. The described behavior is not what happens any more.

Also in the doc string "must be lists of data bjects corresponding"
bjects -> objects